data/human/human missions.txt

```diff
@@ -9685,18 +9683,21 @@ mission "Ice Queen 3"
 			choice
 				`	(Fill out more forms.)`
+			`	A page straight out of an entry exam faces you. Accompanied by a set of dots representing if or how often you have caught them, are dozens of labels noting common diseases, illnesses you thought had been eradicated, and the majority of options consisting of words and names you imagine must represent some serious conditions, but which you have no recollection of ever reading or hearing about. After scraping your memory as best as you can for your medical records, you eventually finish filling out the last set of dots, and sign. Sighing, you flip the form.`
 			choice
 				`	(Continue filling out forms.)`
-			
+			`	You begin to wonder how is it that it was less bureaucratically stressful to purchase your first ship, than it is to schedule an encounter with a campus professor here. When the form prompts you for your address, you scramble out of the room to research what spaceship captains should fill out in such cases. After many contradicting forums and blatantly outdated government websites, you return and put in the best answer you can think of. Then the form specifically requires you to input the institution you studied at, here on <origin>. Again you set to look for answers on how to respond in such cases where you are not even from the planet this form is from. After a few more questions, you start to zone out and just automatically input whatever seems least likely to get the form rejected, based on online tips and nebulous descriptions from government pages.`
 			`	The receptionist brings a few of the earlier forms you filled out and helpfully points out a few mistakes for you to fix.`
 			choice
 				`	(Fix the forms you already filled out.)`
+			`	As it turns out, your bank account is not compatible with the due process the forms here are a part of. You download an application to open your very own account on one of the local banks, after checking with the receptionist that this one definitely is compatible, and after a few more minutes (and a great many attempts at fitting your face in a camera shot for a mandatory facial recognition step) your new bank details are ready and filled in the forms, which you turn over with haste. Surely it must be ending soon.`
 			choice
 				`	(Fill out the final set of forms.)`
-
-			`	A few painfully tedious hours later, all the right forms have been filled out and you're met by an intern in a small vehicle.`
+			`	In this wondrous paradise of terraforming, a true testament of human ingenuity and the acquired mastery over nature, you are close to fuming with frustration at the contrasting inability of the local machinery to recognize your signature due to otherwise imperceptible, millimetric differences between them. For the better part of an hour you get to work on every form again, straining your arms as you slowly, carefully imprint your signature again, on every one of them, as identical to each other as you can.`
```

Comment:
I think it's funnier if you remove this bit and make the sentence a bit snappier.
```suggestion
			`	In this wondrous paradise of terraforming, a true testament to human ingenuity and mastery over nature, you are close to fuming with frustration at the inability of the local machinery to recognize your signature. For the better part of an hour you get to work on every form again, straining your arms as you slowly, carefully imprint your signature again, on every one of them, as identical to each other as you can.`
```